src/bin/am/server/functions.rs

```diff
@@ -0,0 +1,65 @@
+use axum::response::{IntoResponse, Response};
+use axum::Json;
+use http::StatusCode;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+pub(crate) async fn all_functions() -> Result<impl IntoResponse, AllFunctionError> {
+    let functions = am_list::list_all_project_functions(
+        std::env::current_dir()
+            .map_err(|_| AllFunctionError::DirNotFound)?
+            .as_path(),
```

Comment:
It would be better to try to find the root of the project (and fall back to `current_dir` otherwise), because this way there would be more chances to have the correct module name for the detected functions. But it doesn't matter too much if front-end has to do some heuristics to match this list with the list of functions in explorer.

Reply:
for the first version this should be fine enough. we wanted it to follow the same behavior as `am` finding a `am.toml` (aka just current dir)